src/cljc/athens/common_events/resolver.cljc

```diff
@@ -242,6 +273,41 @@
     tx-data))
 
 
+(defmethod resolve-event-to-tx :datascript/unindent-multi
+  [db {:event/keys [args]}]
+  (println "resolver :datascript/unindent-multi args" args)
+  (let [{:keys [uids
+                f-uid]}              args
+        {parent-order :block/order
+         parent-eid   :db/id}        (common-db/get-parent db [:block/uid f-uid])
+        sanitized-uids               (map (comp
```

Comment:
Yeah you are right, I was looking for a reason why I did this because I usually check what should be passed as input and what should be calculated on resolver. I don't know why I am calculating this on resolver, in the case of `indent/multi` I pass `sanitized-uids` as input but not here why?? 🤷🏽‍♂️

Reply:
As long as we ship it correct, it's just part of the process